docker/Dockerfile.perl5.30.cpanfile-snapshot

```diff
@@ -31,9 +31,16 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     apt-get update && \
     apt-get install --no-install-suggests --no-install-recommends -y build-essential libdb-dev libexpat1-dev libicu-dev libperl-dev libpq-dev libssl-dev libxml2-dev zlib1g-dev pkg-config bzip2 ca-certificates libdb5.3 libexpat1 libicu66 libpq5 libssl1.1 libxml2 moreutils perl postgresql-client-12 postgresql-server-dev-12 zlib1g && \
     rm -f /etc/apt/sources.list.d/pgdg.list && \
+    # Install cpanm (helpful with installing other Perl modules)
     curl -sSL https://cpanmin.us | perl - App::cpanminus && \
-    cpanm Carton JSON::XS && \
+    cpanm \
+        # Helpful (beyond installation) with Perl module dependencies
```

Comment:
Maybe "Helpful for installing locked versions of Perl module dependencies"?

Reply:
Thank you. Updated in commit https://github.com/metabrainz/musicbrainz-server/pull/3211/commits/6f103d0b00ff6675f627dfd6e9e2e316393aeef2